packages/web-app-files/src/components/SideBar/Details/FileDetails.vue

```diff
@@ -293,6 +301,10 @@ export default defineComponent({
         )
       }
       await Promise.all(calls.map((p) => p.catch((e) => e)))
+      const lockedUserResult = await clientService.graphAuthenticated.users.getUser(
```

Comment:
I don't know if this is feasible, I don't think non-admin users can call this endpoint